src/Agent.Plugins/PipelineArtifact/PipelineArtifactServer.cs

```diff
@@ -24,6 +24,7 @@ public class PipelineArtifactServer
         public static readonly string ProofNodes = "ProofNodes";
         public const string PipelineArtifactTypeName = "PipelineArtifact";
         public const string BuildArtifactTypeName = "Container";
+        public const string ArtifactSize = "artifactsize";
```

Comment:
Can you please fix merge conflict , I believe these constants are in the file - https://github.com/microsoft/azure-pipelines-agent/blob/master/src/Agent.Plugins/PipelineArtifact/PipelineArtifactConstants.cs

Reply:
Use Uppercase s? artifactSize

Reply:
Rebased on master and fixed the conflicts, thanks!